test/server/paymentProviders/transferwise/index.test.ts

```diff
@@ -151,6 +169,24 @@ describe('paymentMethods.transferwise', () => {
       expect(fundTransfer.called).to.be.true;
       expect(data).to.have.nested.property('fund');
     });
+
+    it('should throw if balance is not enough to cover the transfer', async () => {
+      getBorderlessAccount.resolves({
+        balances: [
+          {
+            currency: 'USD',
+            amount: { value: 0 },
+          },
+        ],
+      });
+
+      try {
+        await transferwise.payExpense(connectedAccount, payoutMethod, expense);
+        throw Error('Unexpected Error');
+      } catch (e) {
+        expect(e.toString()).to.contain("You don't have enough funds");
+      }
+    });
```

Comment:
**[:no_entry_sign: **Blocker**]** This is a nice test, but it has one flow: if the function doesn't throw it won't enter the catch block and it won't see the `expect`, so the test will pass.

An alternative way to check for the exception is to do this:

```suggestion
      const payExpensePromise = transferwise.payExpense(connectedAccount, payoutMethod, expense);
      await expect(payExpensePromise).to.be.eventually.rejectedWith(
        Error,
        "You don't have enough funds"
      );
```

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

Reply:
Actually, the `throw Error('Unexpected Error');` is what is safeguarding that.
But I think your solution is much more elegant. 🎉

Reply:
I indeed missed the `throw Error('Unexpected Error');`, feel free to mark as resolved then.